src/main/groovy/uk/ac/ox/softeng/maurodatamapper/api/restful/client/MauroDataMapperClient.groovy

```diff
@@ -60,28 +60,28 @@ class MauroDataMapperClient implements Closeable {
         this(DEFAULT_CONNECTION_NAME, properties)
     }
 
-    MauroDataMapperClient(String baseUrl, String username, String password) {
-        this(DEFAULT_CONNECTION_NAME, baseUrl, username, password)
+    MauroDataMapperClient(String baseUrl, String username, String password, Boolean insecureTls) {
```

Comment:
Similarly could you also set a default in these constructors, as it's used in existing scripts.